app/grandchallenge/cases/admin.py

```diff
@@ -102,7 +104,35 @@ def algorithm(self, obj):
             return obj.algorithm_result.job.algorithm_image.algorithm
 
 
+class DownloadableFilter(admin.SimpleListFilter):
+    """Allow filtering on downloadable files."""
+
+    title = "Downloadable"
+    parameter_name = "downloadable"
+
+    def lookups(self, request, model_admin):
+        return (("yes", "Yes"),)
+
+    def queryset(self, request, queryset):
+        if self.value() == "yes":
+            return queryset.filter(staged_file_id__isnull=False)
+        return queryset
+
+
+class RawImageFileAdmin(admin.ModelAdmin):
+    list_filter = (DownloadableFilter,)
+    list_display = ("filename", "upload_session", "download")
+    readonly_fields = ("download",)
+
+    def download(self, instance):
+        if not instance.staged_file_id:
+            return
+        return mark_safe(
```

Comment:
I'm always paranoid about using `mark_safe`. I know that it's technically not needed but could we use `format_html` here instead?

Reply:
Moved this to the admin.